scm/driver/gitea/user_test.go

```diff
@@ -20,10 +20,10 @@ func TestUserFind(t *testing.T) {
 	defer gock.Off()
 
 	gock.New("https://try.gitea.io").
-		Get("/api/v1/user").
+		Get("/api/v1/User").
```

Comment:
noticed this was changed from `/api/v1/user` to `/api/v1/User`. Should this be the case or was this a find replace?